lib/utils/getDimension.js

```diff
@@ -11,26 +11,38 @@ const valueParser = require('postcss-value-parser');
  *
  * @param {import('postcss-value-parser').Node} node
  *
- * @returns {string | null}
+ * @returns {{unit: null, number: null} | valueParser.Dimension}
```

Comment:
[suggest] What if also updating the function description?

https://github.com/stylelint/stylelint/blob/788f612b16d42919ddca6552ba486003d0c47dc2/lib/utils/getDimension.js#L8-L10